### ISSUE TITLE
zipimport.ZipImportError: can't find module '_tkinter'

### ISSUE THREAD

**Issue by cuprumtan** [py2exe/py2exe#111]
Hi!
I'm trying to run builded exe on Windows 10 with Python3.9 and got this errors every time:
```
Traceback (most recent call last):
File "zipextimporter.pyc", line 88, in load_module
File "<frozen zipimport>", line 241, in load_module
File "<frozen zipimport>", line 721, in _get_module_code
zipimport.ZipImportError: can't find module '_tkinter'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "<boot hacks>", line 12, in <module>
File "<boot hacks>", line 5, in tk_env_paths
File "zipextimporter.pyc", line 112, in load_module
ImportError: MemoryLoadLibrary failed loading _tkinter.pyd: �� ������ ��������� ������. (126)
Traceback (most recent call last):
File "zipextimporter.pyc", line 88, in load_module
File "<frozen zipimport>", line 241, in load_module
File "<frozen zipimport>", line 721, in _get_module_code
zipimport.ZipImportError: can't find module '_ctypes'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "zipextimporter.pyc", line 88, in load_module
File "<frozen zipimport>", line 259, in load_module
File "ctypes\__init__.pyc", line 8, in <module>
File "zipextimporter.pyc", line 112, in load_module
ImportError: MemoryLoadLibrary failed loading _ctypes.pyd: �� ������ ��������� ������. (126)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
File "boot_common.py", line 46, in <module>
File "zipextimporter.pyc", line 121, in load_module
zipimport.ZipImportError: can't find module ctypes
Traceback (most recent call last):
File "zipextimporter.pyc", line 88, in load_module
File "<frozen zipimport>", line 241, in load_module
File "<frozen zipimport>", line 721, in _get_module_code
zipimport.ZipImportError: can't find module '_ctypes'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "zipextimporter.pyc", line 88, in load_module
File "<frozen zipimport>", line 259, in load_module
File "ctypes\__init__.pyc", line 8, in <module>
File "zipextimporter.pyc", line 112, in load_module
ImportError: MemoryLoadLibrary failed loading _ctypes.pyd: �� ������ ��������� ������. (126)
```

setup.py:
```
import myapp
from setuptools import find_packages, setup
import py2exe


setup(
    packages=find_packages(),
    service=[
        Target(
            name='service',
            description='myapp',
            modules=['service_win32'],
            cmdline_style='pywin32'
        )
    ],
    console=[{'script': 'start.py', 'dest_base': 'dest_base'}],
    options={
        'py2exe': {
            'packages': [
                'myapp',
                'myapp.localplugins',
                'myapp.localplugins.db',
                'myapp.localplugins.system.windows',
            ],
            'bundle_files': 1,
            'dist_dir': 'dist',
            'xref': False,
            'skip_archive': False,
            'ascii': False,
            'compressed': 2,
            'optimize': 2
        },
    },
    zipfile=None
)
```
How can it be fixed?

**Comment by albertosottile** [comment]
Hi, thanks for reporting this. Without the content of your `myapp` package, it is quite difficult for me to help you. Since you have problems with `zipimporter`, I would first verify if your code works with loading the libraries from disk, that is with `bundle_files: 3` and without `zipfile = None`. 

As a matter of fact, `bundle_files <= 2` is currently supported only for standard library packages, and `zipfile = None` is actually not supported at all (it could work but I do not test it nor I usually provide fixes for these kind of issues). 

If your code runs correctly without these two options, then I am afraid you encountered one of the many limitations of `MemoryModule`, for which I am unfortunately unable to help you.

**Comment by cuprumtan** [comment]
I have tried to build exe without `zipfile` and `bundle_files: 1` and got this error:
```
Fatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding
Python runtime state: core initialized
ModuleNotFoundError: No module named 'encodings'
```
So I approached the problem from the other side and noticed the competing libraries of the current Python version and the old one, which, apparently, was incorrectly deleted. After reinstalling Python and py2exe from scratch finally builded exe works.
Thank you very much!
